assets/chezmoi.io/docs/user-guide/password-managers/bitwarden.md

```diff
@@ -6,22 +6,29 @@ function.
 
 Log in to Bitwarden using:
 
-```console
+```shell
```

Comment:
This should be kept as `console`, as it's a console session, not a shell script. Same for the block below.

Reply:
Good to know, thanks for that!